velox/exec/Aggregate.h

```diff
@@ -429,6 +434,14 @@ AggregateRegistrationResult registerAggregateFunction(
     bool registerCompanionFunctions,
     bool overwrite);
 
+AggregateRegistrationResult registerAggregateFunctionMetadata(
```

Comment:
Is there any particular reason to add 'Metadata' suffix to this function name? Can it be just registerAggregateFunction? If not, let's rename to registerAggregateFunctionWithMetadata.

Reply:
Done.